archivist/archivist.py

```diff
@@ -108,16 +111,6 @@ def __init__(self, url, *, auth=None, cert=None, verify=True):
 
         self._cert = cert
 
-        self._assets = None
-        self._events = None
-        self._locations = None
-        self._attachments = None
-
-        self.assets: Optional[_AssetsClient]
-        self.events: Optional[_EventsClient]
-        self.locations: Optional[_LocationsClient]
-        self.attachments: Optional[_AttachmentsClient]
-
```

Comment:
These don't look like they're related to the logging change. Are they intentional/desirable?

Reply:
the original statements setting the attributes to None is a mistake and I am intending to ask Serhiy whether he is OK with this...

Reply:
otherwise _ will revert
_

Reply:
OK, fine then